### DISCUSSION TITLE
Is there an option to move within the flowchart using the minimap?

### DISCUSSION THREAD

**Discussion by Stephen-Almendinger** [xyflow/xyflow#2474] [Q&A]
For instance, if I click and drag on the minimap it would scroll the actual workflow. Right now I don't see anything in the docs that say that it has this feature so I'm just checking if this is an option and if not then it might make for a good feature request.

**Comment by moklick** [comment]
it's on the list https://github.com/wbkd/react-flow/discussions/2166
